.devcontainer/docker-compose.yml

```diff
@@ -26,32 +26,65 @@ services:
     # Overrides default command so things don't shut down after the process ends.
     command: sleep infinity
 
-    # Runs app on the same network as the database container, allows "forwardPorts" in devcontainer.json function.
-    network_mode: service:postgresql
-
     # Uncomment the next line to use a non-root user for all processes.
     user: vscode
 
     # define environment variables
     environment:
+      # Please adapt to package name:
+      MY_MICROSERVICE_CONFIG_YAML: /workspace/.devcontainer/.dev_config.yaml
+      # Used by db migration:
       DB_URL: postgresql://postgres:postgres@postgresql/postgres
-
     # Use "forwardPorts" in **devcontainer.json** to forward an app port locally.
     # (Adding the "ports" property to this file will not forward from a Codespace.)
 
+
+  # Please remove service dependencies that are not needed:
   postgresql:
     image: postgres:latest
     restart: unless-stopped
     volumes:
-      - postgres-data:/var/lib/postgresql/data
+      - postgres_fs:/var/lib/postgresql/data
     environment:
       POSTGRES_USER: postgres
       POSTGRES_DB: postgres
       POSTGRES_PASSWORD: postgres
-
     # Add "forwardPorts": ["5432"] to **devcontainer.json** to forward PostgreSQL locally.
     # (Adding the "ports" property to this file will not forward from a Codespace.)
 
+  mongodb:
+    image: mongo:latest
+    restart: unless-stopped
+    volumes:
+      - mongo_fs:/data/db
+
+  rabbitmq:
+    image: rabbitmq:3-management
+    ports:
+      - 5672:5672
+      - 15672:15672
```

Comment:
Like this, you can directly interact with the rabbitmq broker from the localhost. E.g. access the rabbitmq dashboard. But I have been questioning this practice myself already for some time because it prevents us from running multiple devcontainer environments (that are using the same ports) in parallel. Probably, we can also remove this and e.g. access the rabbitmq dashboard using the simple-browser feature of vscode.